libd-llvm/src/d/llvm/type.d

```diff
@@ -103,7 +104,17 @@ final class TypeGen {
 				return LLVMPointerType(LLVMInt8TypeInContext(llvmCtx), 0);
 		}
 	}
-	
+
+	LLVMTypeRef getSizeT() {
+		static LLVMTypeRef sizeT;
```

Comment:
I don't think it's terrible... getSizeT clarifies intent and caches the TypeRef so it does not need to be computed, all the time.